cobbler/actions/reposync.py

```diff
@@ -133,7 +128,7 @@ def run(self, name: Optional[str] = None, verbose: bool = True):
                 continue
 
             repo_mirror = os.path.join(self.settings.webdir, "repo_mirror")
-            repo_path = os.path.join(repo_mirror, repo.name)
+            repo_path = os.path.join(repo_mirror, repo.name.strip())
```

Comment:
```suggestion
            repo_path = os.path.join(repo_mirror, repo.name)
```

Let's don't strip on every use. Let's do that in the setter of the object name. I don't see a reason why we should allow leading or trailing spaces in object names. I even believe that this is not possible at the moment.